download_filing_docs.R

```diff
@@ -0,0 +1,41 @@
+
+raw_directory <- Sys.getenv("EDGAR_DIR")
+
+library(RPostgreSQL)
+library(dplyr)
+
+pg <- dbConnect(PostgreSQL())
+
+all_files<-tbl(pg, sql("select * from edgar.filing_docs limit 140"))
+files_downloaded<-tbl(pg, sql("select * from edgar.filing_docs_processed limit 120"))
```

Comment:
This won't work. You need to compare `all_files` with the whole table `edgar.filing_docs_processed`. Better to do the anti-join on the database server and just pull in a finite number of rows (e.g., `collect(n=140)`) at the end.